models/ed/R/met2model.ED2.R

```diff
@@ -149,9 +149,7 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
     ## determine GMT adjustment lst <- site$LST_shift[which(site$acro == froot)]
 
     ## extract variables
-    lat  <- eval(parse(text = lat))
-    lon  <- eval(parse(text = lon))
```

Comment:
I have no idea what was going on here -- maybe the idea was to force these to be numbers? Regardless, `eval(parse(...))` is really janky, and the code works just fine without these lines, so I removed them.